### ISSUE TITLE
Example code load.py not working with?

### ISSUE THREAD

**Issue by lukas-vlcek** [elastic/elasticsearch-py#493]
_Notice: I am not a Python developer so I might be just doing something wrong... please bear with me._

I am trying to run example code `load.py` from this repo against ES `5.0.1`. Getting error like `elasticsearch.exceptions.RequestError: TransportError(400, u'illegal_argument_exception', u"Can't specify parent if no parent field has been configured")`:

The following is recreation script:

````
$ uname -a
Darwin Lukass-MacBook-Pro-2.local 16.1.0 Darwin Kernel Version 16.1.0: Thu Oct 13 21:26:57 PDT 2016; root:xnu-3789.21.3~60/RELEASE_X86_64 x86_64

$ curl localhost:9200
{
  "name" : "bVeWMzE",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "XAO9ToBdQjiHIjOO_j1vdw",
  "version" : {
    "number" : "5.0.1",
    "build_hash" : "080bb47",
    "build_date" : "2016-11-11T22:08:49.812Z",
    "build_snapshot" : false,
    "lucene_version" : "6.2.1"
  },
  "tagline" : "You Know, for Search"
}

$ python --version
Python 2.7.12

$ git clone https://github.com/elastic/elasticsearch-py.git
Cloning into 'elasticsearch-py'...
remote: Counting objects: 4491, done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4491 (delta 0), reused 0 (delta 0), pack-reused 4487
Receiving objects: 100% (4491/4491), 1.18 MiB | 1.26 MiB/s, done.
Resolving deltas: 100% (2936/2936), done.
Checking connectivity... done.

$ pip install python-dateutil GitPython
Collecting python-dateutil
  Downloading python_dateutil-2.6.0-py2.py3-none-any.whl (194kB)
    100% |████████████████████████████████| 194kB 2.1MB/s 
Collecting GitPython
  Downloading GitPython-2.1.0-py2.py3-none-any.whl (441kB)
    100% |████████████████████████████████| 450kB 1.9MB/s 
Collecting six>=1.5 (from python-dateutil)
  Downloading six-1.10.0-py2.py3-none-any.whl
Collecting gitdb2>=2.0.0 (from GitPython)
  Downloading gitdb2-2.0.0-py2.py3-none-any.whl (63kB)
    100% |████████████████████████████████| 71kB 4.5MB/s 
Collecting smmap2>=2.0.0 (from gitdb2>=2.0.0->GitPython)
  Downloading smmap2-2.0.1-py2.py3-none-any.whl
Installing collected packages: six, python-dateutil, smmap2, gitdb2, GitPython
Successfully installed GitPython-2.1.0 gitdb2-2.0.0 python-dateutil-2.6.0 six-1.10.0 smmap2-2.0.1
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

$ pip install elasticsearch
Collecting elasticsearch
  Downloading elasticsearch-5.0.1-py2.py3-none-any.whl (57kB)
    100% |████████████████████████████████| 61kB 943kB/s 
Collecting urllib3<2.0,>=1.8 (from elasticsearch)
  Downloading urllib3-1.19.1-py2.py3-none-any.whl (104kB)
    100% |████████████████████████████████| 112kB 2.8MB/s 
Installing collected packages: urllib3, elasticsearch
Successfully installed elasticsearch-5.0.1 urllib3-1.19.1
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

cd elasticsearch-py/example/

$ python load.py
Traceback (most recent call last):
  File "load.py", line 177, in <module>
    load_repo(es)
  File "load.py", line 138, in load_repo
    chunk_size=50 # keep the batch sizes small for appearances only
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 162, in streaming_bulk
    for result in _process_bulk_chunk(client, bulk_actions, raise_on_exception, raise_on_error, **kwargs):
  File "/usr/local/lib/python2.7/site-packages/elasticsearch/helpers/__init__.py", line 91, in _process_bulk_chunk
    raise e
elasticsearch.exceptions.RequestError: TransportError(400, u'illegal_argument_exception', u"Can't specify parent if no parent field has been configured")
````

**Comment by veatch** [comment]
Just FYI, I submitted a fix for this in #497

**Comment by HonzaKral** [comment]
Closed via #497 
Thanks, @veatch!
